components/delegations/IncomingDelegationsTable.tsx

```diff
@@ -0,0 +1,90 @@
+'use client';
+
+import { flexRender, getCoreRowModel, useReactTable } from '@tanstack/react-table';
+import Error from 'components/common/Error';
+import Loader from 'components/common/Loader';
+import type { Delegation } from 'lib/delegate/DelegatePlatform';
+import { useTranslations } from 'next-intl';
+import NoDelegationsFound from './NoDelegationsFound';
+import { incomingColumns } from './columns';
+
+interface Props {
+  delegations: Delegation[];
+  isLoading: boolean;
+  error: Error | null;
+}
+
+const IncomingDelegationsTable = ({ delegations, isLoading, error }: Props) => {
```

Comment:
I recommend using the `Card` and `Table` components (similar to what is used in the tables on the Signatures tab), that should solve some of the visual consistency issues.